### ISSUE TITLE
Converting a model into PyTorch

### ISSUE THREAD

**Issue by p-christ** [araffin/rl-baselines-zoo#31]
Hi,

I'm trying to load your pre-trained DQN Breakout agent into a PyTorch network.  I have figured out how to get the weights transferred over but the pytorch agent is still not able to do well on the game. 

I'm using the exact same env object as in the non-pytorch case which works so that isn't the cause. I think the cause must be that the stable-baselines agent does some pre-processing behind the scenes that I am not aware of. Can anyone help me with some advice on this? 

e.g. at the very least I imagine the stable-baseline agent normalises pixel values to the range 0-1 but I can't find anywhere where this is happening?

**Comment by araffin** [comment]
Hello,

Could you open that issue in stable-baselines repo?
As it does not seem directly related to th rl zoo.
